### PR TITLE
[WIP] Add categorize subsection of treat section.

### PR DESCRIPTION
**Pull request by blengerich**
All comments/edits welcome.

### CODE REVIEW

File: sections/05_treat.md [hunk 1]
```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
 predicts individuals who are most likely to decline during a clinical trial
 and benefit from the treatment.*
```
Comment (agitter) [inline sections/05_treat.md]: We can remove the italic prompt now that you've written this section.

File: sections/05_treat.md [hunk 2]
```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
...
 
+There has been sustained interest in applying deep learning to clinical
+decision making for over two decades. In 1996, Tu
+[@doi:10.1016/S0895-4356(96)00002-9] compared the effectiveness of artificial
+neural networks and logistic regression, questioning whether deep learning
+would replace traditional statistical methods for predicting medical
+outcomes. He posited that while neural networks have several advantages in
```
Comment (agitter) [inline sections/05_treat.md]: "medical outcomes" is broad.  Maybe "medical outcomes, such as..." and give an example or two of what was discussed in this paper?

File: sections/05_treat.md [hunk 3]
```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
...
+representational power, the difficulties in interpretation may limit clinical
+applications. In 2007, Lisboa and Taktak [@doi:10.1016/j.neunet.2005.10.007]
+examined the use of artificial neural networks in medical journals, concluding
+that neural networks provided an increase in benefit to healthcare provision
```
Comment (agitter) [inline sections/05_treat.md]: Increase relative to what?  Other machine learning methods?  Human baselines?

File: sections/05_treat.md [hunk 4]
```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
...
+in 21 of 27 studies.
+
+While significant progress has been made in developing deep learning methods
```
Comment (agitter) [inline sections/05_treat.md]: I like this paragraph a lot, good framing for the rest of the section

File: sections/05_treat.md [hunk 5]
```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
...
+of suggesting clinical decisions. Nemati et al [@doi:10.1109/EMBC.2016.7591355]
+used deep reinforcement learning to optimize a heparin dosing policy for
+intensive care patients. However, because the ideal dosing policy is unknown,
+the model's predictions must be evaluated on counter-factual data. This
+represents a common challenge when bridging the gap between research and
+clinical practice: because the ground-truth is unknown, researchers struggle
+to evaluate model predictions in the absence of interventional data, but
+clinical application is unlikely until the model has been shown to be effective
+. The impressive applications of deep reinforcement learning to other domains
+(@doi:10.1038/nature16961) have relied on knowledge of the underlying
```
Comment (agitter) [inline sections/05_treat.md]: Add the square brackets instead the parentheses so the parser recognizes the reference: `[(@doi:10.1038/nature16961)]`

File: sections/05_treat.md [hunk 6]
```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
...
+rules underlying biological processes could unleash deep learning methods that
+are currently hampered by the difficulties of counter-factual inference.
+
+##### Efficient Clinical trials
+
+A clinical task to deep learning which has been more successfully applied is
+the assignment of patients to clinical trials. Ithapu et al
+[@doi:10.1016/j.jalz.2015.01.010] used a randomized denoising autoenconder to
+learn a multimodal imaging marker that predicts future cognitive and neural
+decline from fluorodeoxyglucose positron emission tomography, amyloid
```
Comment (agitter) [inline sections/05_treat.md]: "positron emission tomography (PET)"

File: sections/05_treat.md [hunk 7]
```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
...
+be directly interpreted. Choi et al [@arxiv:1611.07012] later extended this
+work to take into account the structure of disease ontologies and found that
+the concepts represented by the model were aligned with medical knowledge. Che
+et al [@arxiv:1512.03542] introduced a knowledge-distillation approach which
+used gradient boosted trees to learn interpretable healthcare features from
+trained deep models.
+
+As this challenge of interpretability is common across many domains, there is
+significant interest in developing generic procedures for knowledge extraction
+from deep models. Ribeiro et al [@arxiv:1602.04938] focus on interpreting
```
Comment (agitter) [inline sections/05_treat.md]: Unsurprisingly, interpretability has come up in multiple sections.  I suggest that we keep this paragraph here for now and merge as-is.  We can consider moving it to the Discussion once we review the entire manuscript in the editing phase.  In that case, we could refer to the Discussion section here.

I'm glad you covered #280 and #308.  These had been on my TODO list.

File: sections/05_treat.md [hunk 8]
```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
...
+that the confidence of convolutional neural networks is not reliable; they can
+output confidence scores over 99.99% even for samples that are purely noise.
+Recently, Fong and Vedaldi [@arxiv:1704.03296] provided a framework for
+understanding black box algorithms by perturbing input data. Further work in
+interpreting predictions and understanding the knowledge learned by deep
+neural networks seem necessary for transformative impact in clinical practice.
+
+##### Integrating Deep Learning with Clinical Practice
+
+As deep learning models are difficult to interpret, many current models have
```
Comment (agitter) [inline sections/05_treat.md]: Good job drawing conclusions about the state of the field

**Comment by agitter** [comment]
Thanks again @blengerich, I'm merging this now.

**Comment by blengerich** [comment]
Great, thanks. Let me know if there is another section you would like me to work on.

**Pull request merged**